### PR TITLE
Text-payload

### PR DESCRIPTION
**Pull request by kneeklers**
I never use any of the previous code, just run the file in the new folder is sufficient

### CODE REVIEW

File: Assignment_Textpayload/Assignment.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import streamlit as st
+from PIL import Image
+import numpy as np
+import io
+
+# Helper functions to encode and decode LSB steganography
+def encode_message_in_image(image, message, num_lsb):
```
Comment (ashhssn) [inline Assignment_Textpayload/Assignment.py:7]: Encode/decode function should be called from modules.

**Comment by sitsbstfrnndz** [comment]
Hi Nic

**Review by ashhssn — CHANGES_REQUESTED** [review]
Please do the changes

File: Assignment_Textpayload/Assignment.py (hunk 1 above)
Comment (sitsbstfrnndz) [inline Assignment_Textpayload/Assignment.py:7]: very cool

**Review by sitsbstfrnndz — COMMENTED** [review]
yadayadayada

File: Assignment_Textpayload/Assignment.py (hunk 1 above)
Comment (sitsbstfrnndz) [reply Assignment_Textpayload/Assignment.py:7]: sure

**Review by ashhssn — APPROVED** [review]

**Pull request closed without merging**